### PR TITLE
[INLONG-2161][Feature] Manager support getClusterConfig

### LINKED ISSUE #2161: [Feature][InLong-Manager] Manager support getClusterConfig
### Description

[Feature][InLong-Manager] Manager support getClusterConfig 

### Use case

request URL         /api/inlong/manager/openapi/sort/standalone/getClusterConfig
requset method   GET
request params  ?clusterName=xxx&md5=xxx

example

image

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by imvan**
### Title Name: [INLONG-2161][Feature] Manager support getClusterConfig

 #2161 

### Motivation

getClusterConfig is the interface that **Sort Stand-alone** acquire  id params and sink params from.
Id params is the params of upstream data store such as the topic of kafka and pulsar which Sort Stand-alone consume from.
While sink params is the dispatch params of Sort Stand-alone Sink, which determine which kafka , pulsar or hive cluster the produced data send to. hence it's something like kafka zklist, brokerlist, pulsar serviceUrl, token..

Here is the example that the config be like.

image

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature?  yes 
  - If yes, how is the feature documented?  JavaDocs
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by gosonzhang** [comment]
@imvan, Why are there so many code submissions? 

From the feature description, this modified code should be very small, is there no rebase when submitting the code? 

Please rebase the code first, and then submit it, thanks!

**Comment by imvan** [comment]
> @imvan, Why are there so many code submissions?
> 
> From the feature description, this modified code should be very small, is there no rebase when submitting the code?
> 
> Please rebase the code first, and then submit it, thanks!

**Comment by imvan** [comment]
> 

It's a new interface that **Sort Stand-alone** acquire cluster config including id params and sink params.
Id params is the params of upstream data store such as the topic of kafka and pulsar which Sort Stand-alone consume from.
While sink params is the dispatch type of Sort Stand-alone Sink,  which determine which kafka or pulsar cluster the produced data send to. hence it's something like kafka zklist, brokerlist,  pulsar serviceUrl, token..
Due to the diversity of upstream data store type and downstrem sink type, there must be many tables to maintain those params of different type of sources and sinks.
Most of files are auto generated by Mybatis code generator, such as those xxxMapper.java and xxxMapper.xml
What we need to care about are those xxxxServiceImpl.java , espacially **SortClusterConfigServiceImpl.java** which is the core logic that how to generate a valid **ClusterConfig**

**Comment by wardlican** [comment]
image

"Result: true" and "errorcode" are not reasonable in the protocol. It is suggested to change them to: "msg" and "code"

File: inlong-manager/manager-dao/src/main/resources/mappers/TaskIdParamsKafkaEntityMapper.xml [hunk 1]
```diff
@@ -68,4 +68,138 @@
...
+    <include refid="Base_Column_List" />
+    from task_id_params_kafka
+    where id = #{id,jdbcType=INTEGER}
+  </select>
+  <delete id="deleteByPrimaryKey" parameterType="java.lang.Integer">
+    delete from task_id_params_kafka
+    where id = #{id,jdbcType=INTEGER}
+  </delete>
+  <insert id="insert" parameterType="org.apache.inlong.manager.dao.entity.TaskIdParamsKafkaEntity">
+    insert into task_id_params_kafka (id, parent_name, topic
```
Comment (wardlican) [inline inlong-manager/manager-dao/src/main/resources/mappers/TaskIdParamsKafkaEntityMapper.xml]: need to sort out the format here

File: inlong-manager/manager-dao/src/main/resources/mappers/TaskIdParamsPulsarEntityMapper.xml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    <include refid="Base_Column_List" />
+    from task_id_params_pulsar
+    where id = #{id,jdbcType=INTEGER}
+  </select>
+  <delete id="deleteByPrimaryKey" parameterType="java.lang.Integer">
+    delete from task_id_params_pulsar
+    where id = #{id,jdbcType=INTEGER}
+  </delete>
+  <insert id="insert" parameterType="org.apache.inlong.manager.dao.entity.TaskIdParamsPulsarEntity">
+    insert into task_id_params_pulsar (id, parent_name, topic
```
Comment (wardlican) [inline inlong-manager/manager-dao/src/main/resources/mappers/TaskIdParamsPulsarEntityMapper.xml]: ditto

**Comment by imvan** [comment]
> image
> 
> "result: true" and "errorCode" are not reasonable in the protocol. It is suggested to change them to: "msg" and "code". use return code to judge whether it is successful or not.

Thx @wardlican , very nice suggestion. The fields "result" and "errCode"  has been replaced with "msg" and "code"

**Review by luchunliang — APPROVED** [review]

**Review by wardlican — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

**Pull request closed without merging**